### PR TITLE
modernpro-cv:1.0.0

### PR DESCRIPTION
**Pull request by jxpeng98**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: 
This is a CV template with two different designs, single columns and double columns. This template is integrated with a reference section and bibliography, which is very useful for an academic job application.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/modernpro-cv/1.0.0/template/cv-single.typ [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    (text: "123@example.com", link: "mailto:123@example.com"),
+  ),
+)
+
+// about
+#section[about]
+#descript[#lorem(50)]
+#sectionsep
+#section("Education")
+#education(institution: [#lorem(4)], major: [#lorem(2)], period: "xxxx-xxxx", location: "UK", description: [
```
Comment (elegaanz) [inline packages/preview/modernpro-cv/1.0.0/template/cv-single.typ:24]: I think the `period` argument was renamed to `date`.

Comment (jxpeng98) [reply packages/preview/modernpro-cv/1.0.0/template/cv-single.typ]: Hi,

Many thanks for checking!

I have corrected the arguments in the template and updated it. 

Thanks!

File: packages/preview/modernpro-cv/1.0.0/template/cv-single.typ [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+#twoline-item(
+  entry1: "Project 1",
+  entry2: "Jan 2023",
+  description: [#lorem(40)],
+)
+// Publication
+#section("Publications")
+
++ @quenouille1949approximate
++ @quenouille1949approximate
```
[the remark this answers is not in the document]
Comment (jxpeng98) [reply packages/preview/modernpro-cv/1.0.0/template/cv-single.typ]: Thanks for checking this!

I have corrected this error with a real reference.

**Comment by elegaanz** [comment]
Thank you!

**Pull request merged**